CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+# Version 3.20.0 (2022-04-27)
+## Added
```

Comment:
Since this is going to be a new version of the SDK, can we also include information on this merged PR? 

https://github.com/Labelbox/labelbox-python/pull/540

## Updated
* `NDJsonConverter` now supports Video bounding box annotations. 
    * Note: Currently does not support nested classifications. 
    * Note: Converting an export into Labelbox annotation types, and back to export will result in only keyframe annotations. This is to support correct import format.

cc @msokoloff1  if this wording looks okay

Reply:
yup looks good. Thanks